### PR TITLE
[webui] move tests which fail intermittently to an rspec feature test

### PR DESCRIPTION
**Pull request by evanrolfe**

### CODE REVIEW

**Comment by hennevogel** [comment]
Whats the reasoning? :-)

**Comment by evanrolfe** [comment]
Since this is one of the flickering tests, I thought to try moving it from the old test suite to rspec feature tests to see if that helps.

**Comment by hennevogel** [comment]
k

File: src/api/spec/features/webui/requests_spec.rb [hunk 1]
```diff
@@ -231,14 +231,24 @@ def branch_and_change_sources
...
         expect(page).to have_text("Open review for #{reviewer.login}")
+        expect(page).to have_text("Request 1 (review)")
+        logout
+
+        login reviewer
+        visit request_show_path(1)
+        click_link('review_descision_link_0')
+        fill_in 'review_comment_0', with: 'Ok for the project'
+        click_button 'review_accept_button_0'
+        expect(page).to have_text('Ok for the project')
```
Comment (hennevogel) [inline src/api/spec/features/webui/requests_spec.rb]: Please also test that the `BsRequest` and `Review` change?

File: src/api/spec/features/webui/requests_spec.rb [hunk 2]
```diff
@@ -231,14 +231,24 @@ def branch_and_change_sources
...
+
+      it 'opens a review and accepts it' do
         login submitter
         visit request_show_path(bs_request)
         click_link 'Add a review'
         find(:id, 'review_type').select('User')
         fill_in 'review_user', with: reviewer.login
         click_button 'Ok'
         expect(page).to have_text("Open review for #{reviewer.login}")
+        expect(page).to have_text("Request 1 (review)")
```
Comment (hennevogel) [inline src/api/spec/features/webui/requests_spec.rb]: Please also test that the `Review` is created.

Comment (evanrolfe) [inline src/api/spec/features/webui/requests_spec.rb]: I could add something along these lines to test its been created:
```ruby
expect(Review.all.count).to eq(1)
```

However IMO line 243 is already checking that a review with an id=1 exists doesn't it?

Comment (hennevogel) [inline src/api/spec/features/webui/requests_spec.rb]: Not really. Just that the `BSRequest` is in state *review*

Comment (evanrolfe) [inline src/api/spec/features/webui/requests_spec.rb]: @hennevogel this has been done now (and the other suggestion too)

**Pull request merged**